diffgraph/structured_export.py

```
@@ -0,0 +1,433 @@
+    """
+    Get git diff stats (additions/deletions) for a file.
+
+    Args:
+        file_path: Path to the file
+        diff_args: Git diff arguments (e.g., ['HEAD~1', 'HEAD'])
+
+    Returns:
+        Dictionary with 'additions' and 'deletions' counts
+    """
+    try:
+        cmd = ['git', 'diff', '--numstat'] + diff_args + ['--', file_path]
```

Ruff (0.14.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)